packages/eslint-config-airbnb-base/README.md

```diff
@@ -15,8 +15,8 @@ Our default export contains all of our ESLint rules, including ECMAScript 6+. It
 1. Ensure packages are installed with correct version numbers by running:
   ```sh
   (
-    export PKG=eslint-config-airbnb-base;
-    npm info "$PKG" peerDependencies --json | command sed 's/[\{\},]//g ; s/: /@/g' | xargs npm install --save-dev "$PKG"
+    PKG=eslint-config-airbnb-base
+    npm info "$PKG" peerDependencies --json | sed 's/[\{\},]//g ; s/: /@/g' | xargs npm install --save-dev "$PKG"
```

Comment:
The goal is to achieve the desired results. Whenever possible, all code should be robust against all circumstances, including "the user intentionally changed something they probably shouldn't have" - this holds in JS as well as in shell scripting.

I don't see how it's "obfuscating" at all. I agree it should be on `xargs` however - and I'm ok with it not being on `npm` because that's not a shell built-in.